pom.xml

```diff
@@ -192,6 +192,8 @@
         <module>redis-hbo-provider</module>
         <module>presto-singlestore</module>
         <module>presto-hana</module>
+        <module>presto-ui</module>
```

Comment:
why are we adding this dependency again? It was removed as part of this PR https://github.com/prestodb/presto/pull/22823